Checkbox/Checkbox.vue

```diff
@@ -72,7 +96,11 @@ function handleStateChange() {
 
     <div
       class="checkbox"
-      :class="[`is-${color}`, { 'is-readonly': !editable }]"
+      :class="[
+        `is-${color}`,
+        { 'is-readonly': !editable },
+        `${checkboxClassVisuals}`,
```

Comment:
this will only work for `string` classes but `ClassType` also supports `Arrays` and `Objects`